cmd/utils/flags.go

```diff
@@ -140,6 +140,10 @@ var (
 		Name:  "rinkeby",
 		Usage: "Rinkeby network: pre-configured proof-of-authority test network",
 	}
+	GoerliFlag = cli.BoolFlag{
+		Name:  "goerli",
+		Usage: "Goerli network: pre-configured proof-of-authority test network",
```

Comment:
nit: can you use `Görli` in strings (assuming all terminals support extended ascii in 2019)

Reply:
Fixed!